train.py

```diff
@@ -21,6 +21,12 @@
 from base_trainer import EvalCallback
 from utils import get_last_checkpoint_or_last_model, parse_checkpoint_step, load_check_merging
 from config_parser import parse_config
+import shutil
+from pathlib import Path
+
+from accelerate import Accelerator
+
+accelerator = Accelerator()
```

Comment:
Do we need `gradient_accumulation_steps` here? Like `accelerator = Accelerator(gradient_accumulation_steps=2)` (docs link https://huggingface.co/docs/accelerate/en/package_reference/accelerator#gradient-accumulation)

Reply:
Actually, this import and declaration is redundant. I forgot to delete it. I do not use it, since, HF Trainer initializes accelerate itself.